app/page/template/partials/template-message.htm

```diff
@@ -64,7 +64,7 @@
       <!-- ko if: asset.is_image() -->
         <div class="message-asset-image">
           <div class="image image-loading" data-bind="
-            attr: {'data-uie-visible': $parent.visible},
+            attr: {'data-uie-visible': $parent.visible && !message.is_expired()},
```

Comment:
My hunch is that is must be `attr: {'data-uie-visible': $parent.visible && !message.is_expired()},` to get the correct state if you add the second check as both are observables